dbgpt/storage/vector_store/connector.py

```diff
@@ -96,36 +96,25 @@ def __init__(
             raise Exception(f"Vector store {vector_store_type} not supported")
 
         logger.info(f"VectorStore:{self.connector_class}")
+
+        if vector_store_config is None:
+            vector_store_config = self.config_class()
+
         self._vector_store_type = vector_store_type
         self._embeddings = (
             vector_store_config.embedding_fn if vector_store_config else None
         )
-
         try:
-            if vector_store_config is not None:
-                config: IndexStoreConfig = self.config_class()
```

Comment:
This part of code is used to merge `vector_store_config` into new created `config`, which type is subclass of `IndexStoreConfig`. Removing will cause `llm_client`, `model_name` fields invalidated. The best practice is refractor `VectorStoreConnector` to `IndexStoreConnector`, we hope your contribution in this pr.

Reply:
you are right, i will fix it.